packages/dnb-eufemia/src/components/slider/SliderProvider.tsx

```diff
@@ -0,0 +1,233 @@
+    context?.getTranslation(localProps).Slider,
+    includeValidProps(
+      context?.FormRow,
+
+      /**
+       * Exclude some props
+       */
+      {
+        vertical: null,
+      }
+    ),
+    context?.Slider
+  )
+
+  const [_id] = React.useState(makeUniqueId)
+  if (!allProps.id) {
+    allProps.id = _id
+  }
+
+  const {
+    step,
+    label, // eslint-disable-line
+    label_direction, // eslint-disable-line
+    label_sr_only, // eslint-disable-line
+    status, // eslint-disable-line
+    status_state, // eslint-disable-line
+    status_props, // eslint-disable-line
+    status_no_animation, // eslint-disable-line
+    global_status_id, // eslint-disable-line
+    stretch, // eslint-disable-line
+    suffix, // eslint-disable-line
+    thumb_title: title, // eslint-disable-line
+    subtract_title, // eslint-disable-line
+    add_title, // eslint-disable-line
+    hide_buttons, // eslint-disable-line
+    number_format,
+    skeleton,
+    max, // eslint-disable-line
+    min, // eslint-disable-line
+    disabled,
+    className, // eslint-disable-line
+    id, // eslint-disable-line
+    on_init, // eslint-disable-line
+    on_change,
+    on_drag_start, // eslint-disable-line
+    on_drag_end, // eslint-disable-line
+    vertical: _vertical,
+    reverse: _reverse,
+    value: _value,
+    children: _children, // eslint-disable-line
+
+    ...attributes // Find a DOM element to forwards props too when multi buttons are supported
+  } = allProps
+
+  const [value, setValue] = React.useState(_value)
+  const [thumbState, setThumbState] =
+    React.useState<ThumbStateEnums>('initial')
+  const thumbIndex = React.useRef<number>(-1)
+  const [isVertical] = React.useState(isTrue(_vertical))
+  const [isReverse] = React.useState(
+    isVertical ? !isTrue(_reverse) : isTrue(_reverse)
+  )
+  const isMulti = Array.isArray(value)
+  const setThumbIndex = (index: number) => {
+    if (!isNaN(index)) {
+      thumbIndex.current = index
+    }
+  }
+
+  /**
+   * Deprecated
+   */
+  if (allProps.use_scrollwheel) {
+    warn('use_scrollwheel is not supported anymore!')
+  }
+
+  const handleCurrentIndex = (currentValue: number) => {
+    let currentIndex = null
+
+    if (thumbIndex.current > -1) {
+      currentIndex = thumbIndex.current
+    } else {
+      currentIndex = closestIndex(currentValue, value as Array<number>)
+      setThumbIndex(currentIndex)
+    }
+
+    return currentIndex
+  }
+
+  const emitChange = (
+    event: MouseEvent | TouchEvent,
+    rawValue: number
+  ) => {
+    if (disabled || isTrue(skeleton)) {
+      return
+    }
+
+    const currentValue = roundValue(rawValue, step)
+
+    if (currentValue > -1 && rawValue !== value) {
+      let newValue: ValueTypes = currentValue
+
+      if (isMulti) {
+        const currentIndex = handleCurrentIndex(currentValue)
+
+        newValue = getValueByIndex(
+          currentIndex,
+          currentValue,
+          value as Array<number>
+        )
+      }
```

Comment:
what you think about this suggestion? 

```suggestion
      if (isMulti) {
        const currentIndex = handleCurrentIndex(currentValue)
       
        newValue = [...value]
        newValue.splice(
          currentIndex,
          1,
          currentValue
        )
      }
```